xonsh/built_ins.py

```diff
@@ -692,6 +720,7 @@ def load_builtins(execer=None, config=None, login=False, ctx=None):
     builtins.__xonsh_regexsearch__ = regexsearch
     builtins.__xonsh_glob__ = globpath
     builtins.__xonsh_expand_path__ = expand_path
+    builtins.__xonsh_expand_braces__ = expand_braces
```

Comment:
so my recommendation is to rename the current expandpath to something else, then write a newexpand path that calls both the current expandpath and expand braces